assets/setup-wizard/purpose/index.js

```diff
@@ -17,59 +11,32 @@ import { useState, useEffect } from '@wordpress/element';
 import { useQueryStringRouter } from '../../shared/query-string-router';
 import { useSetupWizardStep } from '../data/use-setup-wizard-step';
 import { H } from '../../shared/components/section';
+import PurposeItem from './purpose-item';
 
 const purposes = [
 	{
-		id: 'share_knowledge',
-		title: __( 'Sell courses and generate income', 'sensei-lms' ),
+		id: 'sell_courses',
+		label: __( 'Sell courses and generate income', 'sensei-lms' ),
 		description: __(
-			'You are a hobbyist interested in sharing your knowledge.',
-			'sensei-lms'
-		),
-	},
-	{
-		id: 'generate_income',
-		title: __( 'Generate income', 'sensei-lms' ),
-		description: __(
-			'You would like to generate additional income for yourself or your business.',
-			'sensei-lms'
-		),
-	},
-	{
-		id: 'promote_business',
-		title: __( 'Promote your business', 'sensei-lms' ),
-		description: __(
-			'You own a business and would like to use online courses to promote it.',
+			'We will install WooCommerce for free.',
```

Comment:
Eventually, we might want to only include these warnings if we're actually going to install WooCommerce/Sensei Certificates (depending on if they are already installed/activated).

Reply:
Makes sense!

I added it to be part of [this issue](https://github.com/Automattic/sensei/issues/5647#issuecomment-1256583753), since it's just a check when we have the information available. Currently, this information is coming through the "features" step, but since we'll use it in 2 different parts, maybe it's worth handling it a little differently.

Anyway, I added it to that issue, so we can think a little more when implementing it. I also already assigned the issue to me. 😉